oauthenticator/__init__.py

```diff
@@ -1,3 +1,4 @@
 from .oauth2 import *
 from .github import *
 from .bitbucket import *
+from .mediawiki import *
```

Comment:
I would just remove it, and leave the name as MWOauthenticator. It shouldn't be imported if it isn't used. I wouldn't even put GitHub and BitBucket here if they weren't already present in the same namespace when it was just `oauthenticator.py`.